### PR TITLE
DEV-1290: Remove message signing tab and fix unbonding

### PR DESCRIPTION
**Pull request by jay-dee7**

### CODE REVIEW

File: src/network/cheqd.ts [hunk 1]
```diff
@@ -279,6 +279,25 @@ export class CheqClient {
...
+	signTxWithStargateSigningClient = async (wallet: CheqWallet, msgs: EncodeObject[]): Promise<Uint8Array> => {
+		const result = await this.getAccount(wallet.getAddress());
+		if (!result) {
+			throw new Error('error getting account');
+		}
+
+		const rawTxn = await this.stargateSigninClient?.sign(
+			wallet.getAddress(),
+			msgs,
+			{ amount: [{ denom: NanoCheqDenom, amount: '12000' }], gas: '1' },
```
Comment (Eengineer1) [inline src/network/cheqd.ts]: @jay-dee7 Shouldn't gas be 0 here?

File: src/network/cheqd.ts [hunk 2]
```diff
@@ -310,6 +329,7 @@ export class CheqClient {
 			if (i === 0) {
 				signDoc = walletSignedDoc;
 			}
+			// @ts-ignore
```
Comment (Eengineer1) [inline src/network/cheqd.ts]: Why to ignore type checking?

File: src/redux/models/identity.ts [hunk 3]
```diff
@@ -1,9 +1,5 @@
 import { createModel } from '@rematch/core';
-import { Credential, IdentityWallet, RootModel } from '../../models';
```
Comment (Eengineer1) [inline src/redux/models/identity.ts:2]: @jay-dee7 Why to remove basic identity helpers? Is importing replaced at another point?

**Review by Eengineer1 — CHANGES_REQUESTED** [review]
- Typo fixed
- Messages tab is removed
- Transaction signing works
- Denomination fixed

Yet to be fixed
- Correct showing of vested vs vesting breakdown.


Great work so far!

File: src/redux/models/identity.ts (hunk 3 above)
Comment (jay-dee7) [reply src/redux/models/identity.ts:2]: these imports are unused so I removed them

File: src/network/cheqd.ts (hunk 1 above)
Comment (jay-dee7) [reply src/network/cheqd.ts]: yeah actually it can be zero, will update that asap

Comment (jay-dee7) [reply src/network/cheqd.ts]: I see, we don't need this method at all so i'm removing it now

File: src/network/cheqd.ts (hunk 2 above)
Comment (jay-dee7) [reply src/network/cheqd.ts]: that was just wrong I guess. Removing the comment now

**Pull request merged**